pkg/verify/verifyconfigmap/store.go

```diff
@@ -104,8 +113,10 @@ func (s *Store) DigestSignatures(ctx context.Context, digest string) ([][]byte,
 
 	var signatures [][]byte
 	for _, cm := range items {
+		klog.V(4).Infof("searching for %s in signature config map %s", prefix, cm.ObjectMeta.Name)
 		for k, v := range cm.BinaryData {
 			if strings.HasPrefix(k, prefix) {
+				klog.V(4).Infof("key %s from signature config map %s matches %s", k, cm.ObjectMeta.Name, digest)
```

Comment:
Log `prefix` here in case it gets incorrectly generated? A better idea would be logging result in `digestToKeyPrefix` too

Reply:
Shouldn't get here if the digest string had incorrect format. Logging prefix is just for info.

Reply:
I'm already logging the prefix a few lines up with `searching for {prefix} in signature config map...`?